code/game/objects/machinery/mortar.dm

```diff
@@ -675,6 +676,12 @@
 	icon_state = "mlrs_rocket"
 	ammo_type = /datum/ammo/mortar/rocket/mlrs
 
+/obj/item/mortal_shell/rocket/mlrs/gas
+	name = "\improper 60mm 'X-50' rocket"
+	desc = "A 60mm rocket loaded with deadly X-50 gas, meant to be used in saturation fire with high scatter."
```

Comment:
Description would not really help the average unga understand what it's all about, i propose:
desc = "A 60mm rocket for the MLRS loaded with deadly X-50 gas that drains plasma and deals damage to any poor soul standing inside it, meant to be used [How you decide it should be, either shock or area suppresion] with high scatter."